lib/parser/entity_parser.js

```diff
@@ -164,6 +164,14 @@ function setEntityNamesOptions(option) {
       entities[entityName][option.name] = option.value;
     }
   });
+  option.excludedNames.forEach((entityName) => {
```

Comment:
There's a better way to do that I think.  Check JSONEntity. This is the class that will be exported to JSON after everything's done.

Reply:
The test will be easier to write. Do you want me to take care of it, or do you have time?

Reply:
That's why I proposed having a look at JSONEntity. Setting a default there might be the best and easiest solution. Don't apologize :) I should thank you for pushing this!

Reply:
I don't think setting the default would be a good solution for this: it would force all users that are using elastic and jdl to add 'searchEngine * with elastisearch' to their existing models (because by default, elastic is generated)